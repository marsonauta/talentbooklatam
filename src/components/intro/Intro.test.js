import React from 'react';
import renderer from 'react-test-renderer';
import Intro from './Intro'

describe("Introduction component", () => {
  test("Matches the snapshot", () => {
    const component = renderer.create(<Intro />).toJSON()
    expect(component).toMatchSnapshot();
  });
});