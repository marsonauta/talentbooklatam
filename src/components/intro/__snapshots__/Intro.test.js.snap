// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Introduction component Matches the snapshot 1`] = `
<section
  class="intro"
>
  <h2>
    Encuentra el perfil que buscas entre los talentosos profesionales con experiencia en 
    <strong>
      LATAM Airlines Digital
    </strong>
    .
  </h2>
  <p>
    Todos se las ingeniaron con sus superpoderes para enfrentar más de 300 casos de uso y así entregarle a los más de 1 millón de usuarios diarios, la mejor experiencia.
  </p>
</section>
`;
